elisp/log4sly.el

```diff
@@ -0,0 +1,149 @@
+;;; -*- Mode: Emacs-Lisp; -*-
+;;;
+;;; Copyright (c) 2012, Max Mikhanosha. All rights reserved.
+;;; Copyright (c) 2021, Hugh Daschbach
+;;;
+;;; This file is licensed to You under the Apache License, Version 2.0
+;;; (the "License"); you may not use this file except in compliance
+;;; with the License.  You may obtain a copy of the License at
+;;; http://www.apache.org/licenses/LICENSE-2.0
+;;;
+;;; Unless required by applicable law or agreed to in writing, software
+;;; distributed under the License is distributed on an "AS IS" BASIS,
+;;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;;; See the License for the specific language governing permissions and
+;;; limitations under the License.
+
+(require 'cl-lib)
+(require 'log4cl)
+(require 'sly)
+
+(declare-function log4slime-highlight-log-message "log4cl.el")
+(declare-function slime-repl-emit "ext:slime-repl.el")
+
+(defvar log4slime-backend)              ; From log4cl.el
```

Comment:
Why are the ones that come from `log4cl` needed? I thought `require` would enforce the compile/load order.

Reply:
Hmm.  I cannot recall where I saw a warning about this.  Perhaps it was a third party linter.

Testing confirm these are not needed.

Removed.